From 7157000000000000000000000000000000000007 Mon Sep 17 00:00:00 2001
From: RistOS Build <build@ristos.org>
Date: Sun, 07 Sep 2026 00:00:00 -0600
Subject: [PATCH] rist: debrand the factory-image flash scripts

`flash-all.sh` and `flash-all.bat` are generated into every factory zip by
device/common/generate-factory-images-common.sh, and both carry a GrapheneOS
banner that this project must not ship.

    generate-factory-images-common.sh:122-126   flash-all.sh heredoc
    generate-factory-images-common.sh:293-297   flash-all.bat heredoc

Replace the GrapheneOS banner in both flash-all heredocs (sh and bat).

Verified with `git apply --check` against device/common in the 2026090600 tree.
---
diff --git a/generate-factory-images-common.sh b/generate-factory-images-common.sh
--- a/generate-factory-images-common.sh
+++ b/generate-factory-images-common.sh
@@ -116,17 +116,19 @@
 #             S:::::S         T:::::T         O::::::O   O::::::O   P::::P
 # SSSSSSS     S:::::S       TT:::::::TT       O:::::::OOO:::::::O PP::::::PP
 # S::::::SSSSSS:::::S       T:::::::::T        OO:::::::::::::OO  P::::::::P
 # S:::::::::::::::SS        T:::::::::T          OO:::::::::OO    P::::::::P
 #  SSSSSSSSSSSSSSS          TTTTTTTTTTT            OOOOOOOOO      PPPPPPPPPP
 #
-# DO NOT EDIT THIS FILE. THE OFFICIAL GRAPHENEOS INSTALL GUIDE WILL NEVER INSTRUCT YOU
-# TO EDIT THIS FILE. IF YOU ARE FOLLOWING A 3RD PARTY GUIDE YOU SHOULD STOP IMMEDIATELY
-# AND START OVER AND FOLLOW THE OFFICIAL INSTALLATION PROCESS ON THE OFFICIAL WEBSITE.
-# USING THE WEB INSTALLER IS STRONGLY RECOMMENDED https://grapheneos.org/install/web
-# THE CLI INSTALLATION PROCESS SHOULD ONLY BE USED BY USERS WITH SPECIFIC NEEDS.
+# DO NOT EDIT THIS FILE. THE RISTOS INSTALL INSTRUCTIONS WILL NEVER TELL YOU TO EDIT
+# IT. IF YOU ARE FOLLOWING A 3RD PARTY GUIDE, STOP AND START OVER USING THE INSTALL.md
+# PUBLISHED WITH THIS IMAGE AT https://dl.ristos.org/ .
+#
+# THIS FLASHES RistOS. IT IS NOT GrapheneOS. GrapheneOS DID NOT BUILD, SIGN OR
+# DISTRIBUTE THIS IMAGE AND CANNOT SUPPORT IT -- DO NOT REPORT ISSUES TO THEM. THERE IS
+# NO WEB INSTALLER FOR RistOS; THIS SCRIPT IS THE SUPPORTED PATH.
 
 # Copyright 2012 The Android Open Source Project
 #
 # Licensed under the Apache License, Version 2.0 (the "License");
 # you may not use this file except in compliance with the License.
 # You may obtain a copy of the License at
@@ -287,17 +289,19 @@
 ::             S:::::S         T:::::T         O::::::O   O::::::O   P::::P
 :: SSSSSSS     S:::::S       TT:::::::TT       O:::::::OOO:::::::O PP::::::PP
 :: S::::::SSSSSS:::::S       T:::::::::T        OO:::::::::::::OO  P::::::::P
 :: S:::::::::::::::SS        T:::::::::T          OO:::::::::OO    P::::::::P
 ::  SSSSSSSSSSSSSSS          TTTTTTTTTTT            OOOOOOOOO      PPPPPPPPPP
 ::
-:: DO NOT EDIT THIS FILE. THE OFFICIAL GRAPHENEOS INSTALL GUIDE WILL NEVER INSTRUCT YOU
-:: TO EDIT THIS FILE. IF YOU ARE FOLLOWING A 3RD PARTY GUIDE YOU SHOULD STOP IMMEDIATELY
-:: AND START OVER AND FOLLOW THE OFFICIAL INSTALLATION PROCESS ON THE OFFICIAL WEBSITE.
-:: USING THE WEB INSTALLER IS STRONGLY RECOMMENDED https://grapheneos.org/install/web
-:: THE CLI INSTALLATION PROCESS SHOULD ONLY BE USED BY USERS WITH SPECIFIC NEEDS.
+:: DO NOT EDIT THIS FILE. THE RISTOS INSTALL INSTRUCTIONS WILL NEVER TELL YOU TO EDIT
+:: IT. IF YOU ARE FOLLOWING A 3RD PARTY GUIDE, STOP AND START OVER USING THE INSTALL.md
+:: PUBLISHED WITH THIS IMAGE AT https://dl.ristos.org/ .
+::
+:: THIS FLASHES RistOS. IT IS NOT GrapheneOS. GrapheneOS DID NOT BUILD, SIGN OR
+:: DISTRIBUTE THIS IMAGE AND CANNOT SUPPORT IT -- DO NOT REPORT ISSUES TO THEM. THERE IS
+:: NO WEB INSTALLER FOR RistOS; THIS SCRIPT IS THE SUPPORTED PATH.
 
 :: Copyright 2012 The Android Open Source Project
 ::
 :: Licensed under the Apache License, Version 2.0 (the "License");
 :: you may not use this file except in compliance with the License.
 :: You may obtain a copy of the License at
